nixos/modules/services/monitoring/smartd.nix

```diff
@@ -20,7 +20,7 @@ let
       ${pkgs.coreutils}/bin/cat << EOF
       From: smartd on ${host} <${nm.sender}>
       To: undisclosed-recipients:;
-      Subject: SMART error on $SMARTD_DEVICESTRING: $SMARTD_FAILTYPE
+      Subject: $SMARTD_SUBJECT
```

Comment:
?

Reply:
smartmontools already provides a subject for notification emails in `$SMARTD_SUBJECT`.
I think we should use it so that the subject stays consistent with other distros.

Should I split that change in another commit?

Reply:
That would be useful. Especially if you could explain the decisions there for posterity.

Reply:
Done.